doc/extensibility.md

```diff
@@ -88,14 +88,24 @@ StreamJsonRpc includes the following `IJsonRpcMessageFormatter` implementations:
     You can contribute your own via `MessagePackFormatter.SetOptions(MessagePackSerializationOptions)`.
     See alternative formatters below.
 
+1. `SystemTextJsonFormatter` - Uses the [`System.Text.Json` library][SystemTextJson] to serialize each
+    JSON-RPC message as UTF-8 encoded JSON. 
+    All RPC method parameters and return types must be serializable by System.Text.Json,
+    with the additional benefit of `DataContract` and `DataMember` attributes being supported by default
```

Comment:
This is an interesting choice considering that STJ serializes by default all public properties without requiring DataContract/Member attributes, this means that an object that is already STJ-serializable (STJ would simply ignore the attributes) would be serialized differently through plain STJ compared to StreamJsonRpc+STJ.

Reply:
The trade-off is a good point, and not one I had fully considered. 
This decision was based on getting existing tests to pass, and to ease migration of JsonMessageFormatter users to SystemTextJsonFormatter. I figure if they have DCS attributes applied to their type, they probably mean for them to be considered. I don't know why a type would have them when honoring them would lead to a bad serialization result.